docs/src/concepts/project.md

```diff
@@ -5,6 +5,8 @@ components of only one type or of different types. A project is created by impor
 file and then exposing the [matrix](./matrix.md) in `default.nix` and shells in `shell.nix` so that it
 works with the standard Nix tools.
 
-To depend on Nedryland in a project, any standard
-[Nix fetcher](https://nixos.org/nixpkgs/manual/#chap-pkgs-fetchers) can be used. More info on how
-to set up a new project can be found in [Declaring The Project](../declare-project.md).
+To depend on Nedryland in a project, any standard [Nix
+fetcher](https://nixos.org/nixpkgs/manual/#chap-pkgs-fetchers) can be used. More info on
+how to set up a new project can be found in [Declaring The
+Project](../declare-project.md). [Niv](https://github.com/nmattia/niv) provides an
+alternative method of pinning and managing a nix dependency.
```

Comment:
Should we promote flakes here and in declare-project.md?

Reply:
Yes, absolutely!